docs/src/pages/guides/deploy.md

```diff
@@ -182,26 +182,30 @@ jobs:
 
 ## Netlify
 
-In your codebase, make sure you have a [`.nvmrc`](https://github.com/nvm-sh/nvm#nvmrc) file with `node v14.15.1` in it.
-
 You can configure your deploy in two ways, via the Netlify website or with the `netlify.toml` file.
 
-With the `netlify.toml` file, add it at the top level of your project with the following settings:
+## `netlify.toml` file
+
+
+Create a new `netlify.toml` file it at the top level of your project repository with the following settings:
 
 ```toml
 [build]
   command = "npm run build"
   publish = "dist"
 ```
 
-Then, set up a new project on [Netlify](https://netlify.com) from your chosen Git provider.
+Push the new `netlify.toml` file up to to your hosted git repository. Then, set up a new project on [Netlify](https://netlify.com) using you project repo.
```

Comment:
One "to" can be removed.

Replace "you" with "your" towards the end of the line.